### DISCUSSION TITLE
SM tube obsolete

### DISCUSSION THREAD

**Discussion by Cootmaster** [smplayer-dev/smtube#32] [General]
ok - read this  "In version 24.5, the smtube button has been removed because smtube is obsolete and it will be replaced with something else in the future."  regarding  the removed button .   is  SMTUBE  21.10.0 the LAST version  , the site isnt updated anymore.   Also is smtube 21.10.0  STILL  working? i put it in  SMPlayer  24.5.0 it installed ok, minus the tool bar button. so looks as its still able to be used, until a new  replacement can be  created at some point in time

**Comment by smplayer-dev** [comment]
Yes, it still works.
